ferrocene/ci/scripts/run-self-test.sh

```diff
@@ -28,12 +28,6 @@ case "$(cat ferrocene/ci/channel)" in
         exit 1
 esac
 
-list_targets() {
```

Comment:
This currently just fetches all targets which we then try to test. This means Darwin will test armv8r eabi, for example. Instead, we'll now set what gets tested explicitly.

This has a side benefit of erroring if one of the artifacts is missing, instead of silently proceeding with success.